webpack.config.ts

```diff
@@ -79,6 +80,16 @@ const config: Configuration = {
 	},
 	module: {
 		rules: [
+			{
+				test: /octicons-react\//,
+				loader: 'string-replace-loader',
+				options: {
+					search: /\n(.*)Icon.defaultProps = {\n\s+className: octicon/g,
```

Comment:
Doesn't `octicon` needs to quoted? Also, some other small suggestions:

```suggestion
					search: /(\w+)Icon\.defaultProps = {\n\s+className: 'octicon'/g,
```

Reply:
Heh you’re right. The linter suggested this change 🤷‍♂️